release/publish_release.sh

```diff
@@ -220,8 +220,8 @@ if [[ -n "${GCS_SOURCE}" ]]; then
     gsutil -m cp "gs://${GCS_SOURCE}/manifest.xml" "${UPLOAD_DIR}/"
   fi
   if [[ "${DO_GITHUB_REL}" == "true" ]]; then
-    gsutil -m cp "gs://${GCS_SOURCE}/istio*.zip" "${UPLOAD_DIR}/"
-    gsutil -m cp "gs://${GCS_SOURCE}/istio*.gz"  "${UPLOAD_DIR}/"
+    gsutil -m cp "gs://${GCS_SOURCE}/docker.io/istio*.zip" "${UPLOAD_DIR}/"
```

Comment:
Now that the "-" and "_" has been made uniform it could be worthwhile to add a "-" after "istio" in case somebody adds more files that they didn't intend to get published to github.  There's a similar story for the "for FILE in ${2}/istio*.${4}" line in release/create_github_release.sh.